recipes/nolearn/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "nolearn" %}
+{% set version = "0.6.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "bab2dc2aeb9560264791298f1d206b78fd9b345cecdd500b34f469d367765158" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  skip: true  # [win or py!=36]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - joblib
+    - scikit-learn
+    - tabulate
+    - lasagne
+    - theano
+  run:
+    - python
+    - joblib
+    - scikit-learn
```

Comment:
Can you replace this line with `scikit-learn <0.18` ?

Reply:
Nope. I wonder if the problem is ultimately due to the fact that theano doesn't work for py37, which is why lasagne doesn't work for py37, which is forcing me to use py==36 for this recipe